sonic_platform_base/sonic_xcvr/api/public/c_cmis.py

```diff
@@ -388,362 +388,53 @@ def get_transceiver_bulk_status(self):
-        esnrlowwarning               = FLOAT                            ; esnr low warning threshold in db
-        cfohighalarm                 = FLOAT                            ; cfo high alarm threshold in MHz
-        cfolowalarm                  = FLOAT                            ; cfo low alarm threshold in MHz
-        cfohighwarning               = FLOAT                            ; cfo high warning threshold in MHz
-        cfolowwarning                = FLOAT                            ; cfo low warning threshold in MHz
-        txcurrpowerhighalarm         = FLOAT                            ; txcurrpower high alarm threshold in dbm
-        txcurrpowerlowalarm          = FLOAT                            ; txcurrpower low alarm threshold in dbm
-        txcurrpowerhighwarning       = FLOAT                            ; txcurrpower high warning threshold in dbm
-        txcurrpowerlowwarning        = FLOAT                            ; txcurrpower low warning threshold in dbm
-        rxtotpowerhighalarm          = FLOAT                            ; rxtotpower high alarm threshold in dbm
-        rxtotpowerlowalarm           = FLOAT                            ; rxtotpower low alarm threshold in dbm
-        rxtotpowerhighwarning        = FLOAT                            ; rxtotpower high warning threshold in dbm
-        rxtotpowerlowwarning         = FLOAT                            ; rxtotpower low warning threshold in dbm
-        rxsigpowerhighalarm          = FLOAT                            ; rxsigpower high alarm threshold in dbm
-        rxsigpowerlowalarm           = FLOAT                            ; rxsigpower low alarm threshold in dbm
-        rxsigpowerhighwarning        = FLOAT                            ; rxsigpower high warning threshold in dbm
-        rxsigpowerlowwarning         = FLOAT                            ; rxsigpower low warning threshold in dbm
-        ========================================================================
+           Dictionary
         """
-        trans_dom_th = super(CCmisApi,self).get_transceiver_threshold_info()
-
-        for vdm_key, trans_dom_th_key_prefix in C_CMIS_DELTA_VDM_KEY_TO_DB_PREFIX_KEY_MAP.items():
-            for i in range(1, 5):
-                trans_dom_th_key = trans_dom_th_key_prefix + VDM_SUBTYPE_IDX_MAP[i]
-                self._update_dict_if_vdm_key_exists(trans_dom_th, trans_dom_th_key, vdm_key, i)
+        trans_status = super(CCmisApi,self).get_transceiver_status()
+        trans_status['tuning_in_progress'] = self.get_tuning_in_progress()
+        trans_status['wavelength_unlock_status'] = self.get_wavelength_unlocked()
 
-        return trans_dom_th
+        return trans_status
 
-    def get_transceiver_status(self):
+    def get_transceiver_status_flags(self):
         """
-        Retrieves transceiver status of this SFP
+        Retrieves transceiver status flags for this SFP module.
```

Comment:
@mihirpat1 SFP -> Active modules?

Reply:
@prgeor This API can is also used for non-active modules. Hence, planning to keep this as is per our discussion.